server/src/main/java/jass/server/util/GameUtil.java

```diff
@@ -332,6 +363,246 @@ public void onPlayedCard(final PlayCardData data) throws InterruptedException {
         }
     }
 
+    ////////////////////////////////////////////////////////////////////////////
+    // Figure out the points for cards.
+    ////////////////////////////////////////////////////////////////////////////
+
+    /**
+     * @param cards All cards
+     *
+     * @return Returns the points of all cards together.
+     *
+     * @author Manuele Vaccari
+     */
+    private static int calculateCardPointsTrumpf(final CardEntity... cards) {
+        int points = 0;
+        for (CardEntity card : cards) {
+            points += card.getRank().getPointsTrumpf();
+        }
+        return points;
+    }
+
+    /**
+     * @param cards All cards
+     *
+     * @return Returns the points of all cards together.
+     *
+     * @author Manuele Vaccari
+     */
+    private static int calculateCardPointsObeAbe(final CardEntity... cards) {
+        int points = 0;
+        for (CardEntity card : cards) {
+            points += card.getRank().getPointsObeAbe();
+        }
+        return points;
+    }
+
+    /**
+     * @param cards All cards
+     *
+     * @return Returns the points of all cards together.
+     *
+     * @author Manuele Vaccari
+     */
+    private static int calculateCardPointsOndeUfe(final CardEntity... cards) {
+        int points = 0;
+        for (CardEntity card : cards) {
+            points += card.getRank().getPointsOndeufe();
+        }
+        return points;
+    }
+
+    ////////////////////////////////////////////////////////////////////////////
+    // Figure out the winner of a turn.
+    ////////////////////////////////////////////////////////////////////////////
+
+    /**
+     * @param currentRound The current round.
+     * @param currentTurn  The current turn.
+     *
+     * @return Returns the user who won the current turn.
+     *
+     * @author Manuele Vaccari
+     */
+    private static UserEntity validateTurnWinner(final RoundEntity currentRound, final TurnEntity currentTurn) {
```

Comment:
the method naming is confusing. It's not validating the move, but calculating who won. -> calculateTurnWinner